dao/pgdb.go

```diff
@@ -169,7 +169,7 @@ func Invalid() bool {
 func (p *PgdbConn) SwitchSchema(ctx Context, name string) error {
 	return ExecDB(ctx,
 		SQLExecf("CREATE SCHEMA IF NOT EXISTS %q", name),
-		SQLExecf("SET search_path TO %q", name),
+		SQLExecf("SET search_path TO public, %q", name),
```

Comment:
I don't think it's right. This will create all schema under public. Should be another way.

Reply:
We can try adding:
```go
SQLExecf("create extension if not exists postgis") // after line 172
```
Or rearrange:
```go
SQLExecf("SET search_path TO %q, public", name),
```

Reply:
it sets search_path to use public additionally to our custom schema. the Logic for dedicated schema creation is not changed.  

It doesn't hurt, and allows to avoid adding `public.` in our queries.

Reply:
hmm, tried that create extension, doesn't work.. as the extension already exists in public.  
search_path option is working, let's keep that for now.

Reply:
Change it to
```go
SQLExecf("SET search_path TO %q, public", name)
```
Otherwise all tables will be created in public schema by default.